Automattic/jetpack
# Carousel: Visually indicate that an image has comments

Fixes https://github.com/Automattic/view-design/issues/312

#### Changes proposed in this Pull Request:

This PR adds a visual indicator plus comment count to communicate that a carousel image has comments.


**Dark theme**

<img width="485" alt="Screen Shot 2021-06-29 at 11 47 06 am" src="https://user-images.githubusercontent.com/6458278/123724526-d64afa80-d8cf-11eb-9cb4-7059b15e03be.png">


**Light theme**

<img width="485" alt="Screen Shot 2021-06-29 at 11 45 08 am" src="https://user-images.githubusercontent.com/6458278/123724519-d3e8a080-d8cf-11eb-86bf-7e0d75a9524d.png">


#### Jetpack product discussion

Motivation p58i-aLm-p2
Design p9Jlb4-2cI-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Create a tiled gallery, making sure the carousel is activated in `Jetpack Settings -> Writing`
2. Notice the comment icon on each gallery image slide
3. Comment on an image and refresh the page
4. You should see the comment indicator on the slides that have comments. Make sure the comment count corresponds with the number of comments.
